cmd/builder/.goreleaser.yml

```diff
@@ -8,6 +8,7 @@ version: 2
 builds:
   - flags:
       - -trimpath
+      - -buildmode=pie
```

Comment:
this change is safe to make because we only currently build/release ocb for platforms that support Internal Linking (buildmode=pie is successful even when CGO is disabled)